test/src/test/java/org/corfudb/AbstractCorfuTest.java

```diff
@@ -95,18 +102,43 @@ protected void succeeded(Description description) {
                     .reset().a("]" + testStatus).newline());
         }
 
-        /** Run when the test fails.
+        /** Run when the test fails, prints the name of the exception
+         * with the line number the exception was caused on.
          * @param e             The exception which caused the error.
          * @param description   A description of the method run.
          */
         protected void failed(Throwable e, Description description) {
             System.out.print(ansi().a("[")
                     .fg(Ansi.Color.RED)
-                        .a("FAIL - ").reset()
-                    .a(e.getClass().toString())
+                        .a("FAIL").reset()
+                    .a(" - ").a(shortClassName(e.getClass()))
+                    .a(":L").a(getLineNumber(e, description))
                     .a("]").newline());
         }
 
+        /** Get the short class name from the fully qualified class.
+         * @param cls   The class to get the name of.
+         * @return      The short class name.
+         */
+        private String shortClassName(Class<?> cls) {
+            int lastIndex = cls.toString().lastIndexOf('.');
```

Comment:
Can you update this, so I can pull this in and then rebase annotationProcessor against master?